BrightID/src/components/Apps/appThunks.ts

```diff
@@ -417,24 +413,13 @@ export const requestSponsoring =
     dispatch(
```

Comment:
Looks like we don't need the state SPONSOR_PRECHECK_APP anymore, as the operation `api.sponsor` is now submitted without checking if it was submitted already before. 
If my understanding is correct we can delete the state SPONSOR_PRECHECK_APP from the app_linking_steps enum. Transition can go directly from `PRELINK_CHECK_PASSED` to `SPONSOR_WAITING_OP`.